src/plugins/orangehrmRecruitmentPlugin/config/routes.yaml

```
@@ -105,6 +105,36 @@ apiv2_recrutiment_candidates_shortlist:
     _api: OrangeHRM\Recruitment\Api\CandidateShortlistAPI
     _key: id
 
+apiv2_recrutiment_candidates_schedule_interview:
+  path: /api/v2/recruitment/candidates/{candidateId}/shedule-interview
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ POST ]
+  defaults:
+    _api: OrangeHRM\Recruitment\Api\CandidateScheduleInterviewAPI
+    _key: id
+
+apiv2_recruitment_interviews_attachment:
+  path: /api/v2/recruitment/interviews/{interviewId}/attachments/{attachmentId}
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET, PUT ]
+  defaults:
+    _api: OrangeHRM\Recruitment\Api\InterviewAttachmentAPI
+    _key: attachmentId
+
+apiv2_recruitment_interview_attachments:
+  path: /api/v2/recruitment/interviews/{interviewId}/attachments
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET ]
+  defaults:
+    _api: OrangeHRM\Recruitment\Api\InterviewAttachmentAPI
+
+apiv2_recruitment_interviews_attachments:
+  path: /api/v2/recruitment/interviews/attachments
```

```suggestion
  path: /api/v2/recruitment/interviews/{interviewId}/attachments
```
can't we use this way. you can merge `apiv2_recruitment_interviews_attachments` and  `apiv2_recruitment_interview_attachments`.
there is no restriction to pass all data in request body. can  remove `interviewId` from request body and get from attribute

---

yes, refactored.